tokio/src/lib.rs

```diff
@@ -19,6 +19,7 @@
 #![cfg_attr(docsrs, allow(unused_attributes))]
 #![cfg_attr(loom, allow(dead_code, unreachable_pub))]
 #![cfg_attr(windows, allow(rustdoc::broken_intra_doc_links))]
+#![cfg_attr(all(target_os = "wasi", target_env = "p2"), feature(wasip2))]
```

Comment:
This doesn't seem like the right thing to do:

https://github.com/rust-lang/rust/issues/130323#issuecomment-2350379144